examples/tutorials/translations/german/Part 01 - The Basic Tools of Private Deep Learning.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<h1>Herzliche Glückwünsche!!! - Zeit, der Community beizutreten!&nbsp;</h1><p> </p><p>Herzlichen Glückwunsch zum Abschluss dieses Notebook-Tutorials! Wenn es Ihnen gefallen hat und Sie sich der Bewegung zur Wahrung der Privatsphäre, zum dezentralisiertenen Besitz von KI und der KI-Lieferkette (Daten) anschließen möchten, können Sie dies auf folgende Weise tun!&nbsp;</p><h3><br></h3><h3>PySyft auf GitHub einen Stern geben!&nbsp;</h3><p> </p><p>Der einfachste Weg, unserer Community zu helfen, besteht darin, die GitHub-Repos mit Sternen auszuzeichnen! Dies hilft, das Bewusstsein für die coolen Tools zu schärfen, die wir bauen.&nbsp;</p><p> </p><p><a href="https://github.com/OpenMined/PySyft" target="_blank">Gib PySyft einen Stern</a></p><p> </p><h3>Mach mit bei Slack!&nbsp;</h3><p> </p><p>Der beste Weg, um über die neuesten Entwicklungen auf dem Laufenden zu bleiben, ist, sich unserer Community anzuschließen! Sie können dies tun, indem Sie das Formular unter&nbsp;<a href="http://slack.openmined.org/" target="_blank">http://slack.openmined.org</a>&nbsp;ausfüllen.</p><p>&nbsp;</p><h3>Treten Sie einem Code-Projekt bei!&nbsp;</h3><p> </p><p>Der beste Weg, um zu unserer Community beizutragen, besteht darin, Entwickler zu werden! Sie können jederzeit zur PySyft GitHub Issues-Seite gehen und nach "Projects" filtern. Dies zeigt Ihnen alle Top-Level-Tickets und gibt einen Überblick darüber, an welchen Projekten Sie teilnehmen können! Wenn Sie nicht an einem Projekt teilnehmen möchten, aber ein wenig programmieren möchten, können Sie auch nach weiteren "einmaligen" Miniprojekten suchen, indem Sie nach GitHub-Problemen suchen, die als "good first issue" gekennzeichnet sind.&nbsp;</p><ul><li><a href="https://github.com/OpenMined/PySyft/issues?q=is%3Aopen+is%3Aissue+label%3AProject" target="_blank">PySyft Projects</a></li><li><a href="https://github.com/OpenMined/PySyft/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22" target="_blank">Good First Issue Tickets</a></li></ul><h3>Spenden&nbsp;</h3><p> </p><p>Wenn Sie keine Zeit haben, zu unserer <em>Codebase</em> beizutragen, aber dennoch Unterstützung leisten möchten, können Sie auch Unterstützer unseres Open Collective werden. Alle Spenden fließen in unser Webhosting und andere Community-Ausgaben wie Hackathons und Meetups!&nbsp;</p><p><a href="https://opencollective.com/openmined" target="_blank">OpenMined's Open Collective Page</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/3178/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>